mailbox/store/src/test/java/org/apache/james/mailbox/store/StoreMailboxMessageResultIteratorTest.java

```diff
@@ -99,6 +101,11 @@ public MailboxCounters getMailboxCounters(Mailbox mailbox) throws MailboxExcepti
                 .build();
         }
 
+        @Override
+        public List<MailboxCounters> getMailboxCounters(Collection<MailboxId> mailboxIds) throws MailboxException {
+            throw new NotImplementedException("sorry");
```

Comment:
I think it deserves a longer comment

Reply:
We should use UnsupportedOperationException as in other unsuported method this class provides.